metron-deployment/README.md

```diff
@@ -1,175 +1,129 @@
+
+#### What does this do?
+This builds installable RPM packages that allow you to install Metron on an RPM-based operating system like CentOS.
+
+#### How do I do it?
 ```
 cd metron-deployment
 mvn clean package -Pbuild-rpms
 ```
 
-The output RPM files will land in `target/RPMS/noarch`.  They can be installed with the standard
-```
-rpm -i <package>
-```
+The RPM packages will land in `metron-deployment/packaging/docker/rpm-docker/RPMS/noarch`.   
+
+#### Where can I get more information?
+
+* [RPM Docker](packaging/docker/rpm-docker)
 
-## Kibana Dashboards
+## How do I build DEB packages?
 
-The dashboards installed by the Kibana custom action are managed by the dashboard.p file.  This file is created by exporting existing dashboards from a running Kibana instance.
+**WARNING**: The DEB packages are a recent addition to Metron.  These packages have not undergone the same level of testing as the RPM packages.  Improvements and more rigerous testing of these packages is underway and will improve in future releases.  Until then, use these at your own risk.
 
-To create a new version of the file, make any necessary changes to Kibana (e.g. on full-dev), and export with the appropriate script.
+#### Who is this for?
+If you want to manually install Metron on a APT-based system like Ubuntu or if you want a guided installation process using the Ambari Mpack on an APT-based system, then this is the right option for you.
 
+#### What does this do?
+This builds installable DEB packages that allow you to install Metron on an APT-based operating system like Ubuntu.
+
+#### How do I do it?
 ```
-python packaging/ambari/metron-mpack/src/main/resources/common-services/KIBANA/4.5.1/package/scripts/dashboard/dashboardindex.py \
-$ES_HOST 9200 \
-packaging/ambari/metron-mpack/src/main/resources/common-services/KIBANA/4.5.1/package/scripts/dashboard/dashboard.p -s
+cd metron-deployment
+mvn clean package -Pbuild-debs
 ```
 
-Build the Ambari Mpack to get the dashboard updated appropriately.
+The DEB packages will land in `metron-deployment/packaging/docker/deb-docker/target`.  
+
+#### Where can I get more information?
+* [DEB Docker](packaging/docker/deb-docker)
+
 
-Once the MPack is installed, run the Kibana service's action "Load Template" to install dashboards.  This will completely overwrite the .kibana in Elasticsearch, so use with caution.
+## How do I deploy Metron within AWS?
 
-## Kerberos
-The MPack can allow Metron to be installed and then Kerberized, or installed on top of an already Kerberized cluster.  This is done through Ambari's standard Kerberization setup.
+**WARNING**: This is only intended for short-lived testing and development.  This deployment method has the following severe limitations.
+* The cluster is not secured in any way. It is up to you to manually secure it.  
+* The cluster will not survive a reboot.
 
-### Caveats
```

Comment:
Some of the caveats info doesn't exist anywhere else (to the best of my knowledge).  I don't think this is all obsolete, should this be either integrated elsewhere, or be added back in (at least in some form)?

Reply:
I will create a README in the metron-mpack/ directory and move all those details there.

Reply:
Do you think that would work?  I was trying to make the metron-deployment/README just a landing pad that directs users to the appropriate bits without getting bogged down in technical details.

Reply:
I think that would work, but I think it's worth linking to it from here.  This'll be the first place people look for these sort of things, and giving a pointer to something a bit more in the weeds like the mpack stuff would probably be helpful

Reply:
Yes, definitely.  The intention is to link to it from here.

Reply:
Ready for review @justinleet